admin/tools/docker/tagged/Dockerfile.tpl

```diff
@@ -3,16 +3,17 @@ MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
 
 USER root
 
-RUN apt-get update && apt-get -y install byobu git vim gdb lsof
+RUN apt-get update
```

Comment:
ok, I do this. But note it is sometime convenient to have 2 layers for debugging purpose.